Makefile

```diff
@@ -1,4 +1,11 @@
-.PHONY: test
+.PHONY: test lint
```

Comment:
nit: can you keep this as

```
.PHONY: test
test:
```

I prefer to keep the .PHONY blocks above the rule they apply to.

Reply:
Wasn't planned this way should have been:

```
.PHONY: test
test: lint
```

similar to the new build task. I've updated the file.